runtime/util/ConfigurationParser.cpp

```diff
@@ -33,6 +33,10 @@ uint32_t getTagForSymbolName(const char *name) {
     return lb->second;
   }
   uint32_t const tag = getTagForSymbolNameInternal(s.c_str());
+  assert(
```

Comment:
let's not use assert for this, as it will not trigger if NDEBUG is set, as it generally is when we are building release builds of semantics.

Reply:
Agreed - it would be nice to print the symbol name as well, given that it's available.

Reply:
Agreed. I just pushed a change that prints the error message to `stderr` and aborts.